apis/securesourcemanager/v1alpha1/repository_types.go

```diff
@@ -47,6 +47,15 @@ type SecureSourceManagerRepositorySpec struct {
 
 	// Input only. Initial configurations for the repository.
 	InitialConfig *Repository_InitialConfig `json:"initialConfig,omitempty"`
+
+	// Optional. Description of the repository, which cannot exceed 500 characters.
+	// Temporarily omitted for now.
+	// Description *string `json:"description,omitempty"`
+
+	// Optional. This checksum is computed by the server based on the value of other
+	// fields, and may be sent on update and delete requests to ensure the
+	// client has an up-to-date value before proceeding.
+	Etag *string `json:"etag,omitempty"`
```

Comment:
Per the description of this field, I feel this is a computed field(output-only), and need to be added to ObservedState?